src/codecoverage/bot/tests/test_zero_coverage.py

```diff
@@ -8,6 +8,75 @@
 from code_coverage_bot.zero_coverage import ZeroCov
 
 
+def test_zero_coverage_some_platform(tmpdir,
```

Comment:
The test is checking incorrectly that `mozglue/build/dummy.cpp` is in the report. This file is only present in the Windows report, so it is one of those files conditionally built. Same with `toolkit/components/osfile/osfile-win.jsm`, this file is only in the Linux report.

You should add a couple new fixture files similar to `jsvm_artifact` and `grcov_artifact` where you have the source files covered in one report and uncovered in another report.

Then, you'll assert that `mozglue/build/dummy.cpp` and `toolkit/components/osfile/osfile-win.jsm` are not present in the final report, but the files you added are.

Feel free to ask if you need more details.